src/common/resourcemonitor/resourcemonitor.cpp

```diff
@@ -16,8 +16,8 @@ namespace monitoring {
  * Public
  **********************************************************************************************************************/
 
-Error ResourceMonitor::Init(
-    ResourceUsageProviderItf& resourceUsageProvider, SenderItf& monitorSender, aos::SubscribableItf& subscribable)
+Error ResourceMonitor::Init(ResourceUsageProviderItf& resourceUsageProvider, SenderItf& monitorSender,
+    aos::ConnectionPublisherItf& subscribable)
```

Comment:
it is in `aos` namespace. Rename `subscribable` to `connectionPublisher` or so on. We may have some other publishers in the future.

Reply:
Done

Reply:
Done